src/jsonUtils/hacksForJSONImpl.js

```diff
@@ -95,6 +159,49 @@ export const makeImageDataFromUrl = (url: string): MSImageData => {
   return MSImageData.alloc().initWithImage_convertColorSpace(image, false);
 };
 
+export function makeResizeConstraint(
+  resizingConstraint: ?ResizeConstraints
+): number {
+  if (resizingConstraint) {
+    const constraints = [];
+    const {
+      top,
+      right,
+      bottom,
+      left,
+      fixedHeight,
+      fixedWidth,
+    } = resizingConstraint;
+
+    if (top) {
+      constraints.push('top');
+    }
+    if (right) {
+      constraints.push('right');
+    }
+    if (bottom) {
+      constraints.push('bottom');
+    }
+    if (left) {
+      constraints.push('left');
+    }
+    if (fixedHeight) {
+      constraints.push('fixedHeight');
+    }
+    if (fixedWidth) {
+      constraints.push('fixedWidth');
+    }
+
+    if (constraints.length > 0) {
+      return RESIZE_CONSTRAINTS[constraints.join('_')];
+    }
+
+    return RESIZE_CONSTRAINTS.none; // No constraints
```

Comment:
why not let this fall through to the outer `return`?

Reply:
Good call! Removed.